.travis.yml

```diff
@@ -1,43 +1,40 @@
-sudo: false
+sudo: required
```

Comment:
Wait... how did it work before with `sudo: false`? I didn't look closely enough earlier to notice it was running `dpkg`. The need for sudo is why I didn't mess with this earlier; I'd rather not lose that. It looks like it is possible [to install into the home directory though](https://docs.bazel.build/versions/master/install-ubuntu.html).

Reply:
I don't know why...